test/main.js

```diff
@@ -24,6 +26,8 @@ var SelfInsertingView = View.extend({
     render: function () {}
 });
 
+viewCompliance.view(test, ViewSwitcher, { view: new makeTestView()() });
```

Comment:
Additionally, I might simplify the view compliance line by simply passing in the `el` prop to the options.

viewCompliance.view(test, ViewSwitcher, { el: document.createElement("div") });